server/src/main/java/io/seata/server/session/GlobalSession.java

```diff
@@ -58,6 +62,165 @@
 
     private ArrayList<BranchSession> branchSessions = new ArrayList<>();
 
+    protected StateGraph stateGraph = new StateGraph();
```

Comment:
how ablout init it in static ？because it seems to apply to all GlobalSession,and  the stageGraph rule can't change dynamically.

Reply:
ok，thanks for your advice。done